packages/mobile/src/recipients/recipient.ts

```diff
@@ -127,21 +124,22 @@ export interface RecipientInfo {
   addressToE164Number: AddressToE164NumberType
   phoneRecipientCache: NumberToRecipient
   valoraRecipientCache: AddressToRecipient
-  // this info comes from Firebase for known addresses (ex. Simplex, cUSD incentive programs)
-  // differentiated from valoraRecipients because they are not displayed in the RecipientPicker
-  addressToDisplayName: AddressToDisplayNameType
 }
 
-export function getRecipientFromAddress(address: string, info: RecipientInfo) {
+export function getRecipientFromAddress(
+  address: string,
+  info: RecipientInfo,
+  defaultName?: Maybe<string>,
+  defaultImage?: Maybe<string>
+): Recipient {
   const e164PhoneNumber = info.addressToE164Number[address]
   const numberRecipient = e164PhoneNumber ? info.phoneRecipientCache[e164PhoneNumber] : undefined
   const valoraRecipient = info.valoraRecipientCache[address]
-  const displayInfo = info.addressToDisplayName[address]
 
   const recipient: Recipient = {
     address,
-    name: valoraRecipient?.name || numberRecipient?.name || displayInfo?.name,
-    thumbnailPath: valoraRecipient?.thumbnailPath || displayInfo?.imageUrl || undefined,
```

Comment:
We need to keep `addressToDisplayName` for correctly showing Bidali transactions.

This makes me think we likely want a migration to only keep Bidali addresses [in the identify reducer](https://github.com/valora-inc/wallet/blob/cdf364a68b3745de20b7357dcf3ce8b14449fbe4/packages/mobile/src/identity/reducer.ts#L100).

Reply:
Oh yeah good idea!